rss2email/feed.py

```diff
@@ -404,9 +404,12 @@ def _process(self, parsed):
 
     def _check_for_errors(self, parsed):
         warned = False
-        status = getattr(parsed, 'status', 200)
+        status = getattr(parsed, 'status', None)
         _LOG.debug('HTTP status {}'.format(status))
-        if status == 301:
+        if status is None:
+            # On non-HTTP schemes, like file://. See commit message for further details.
```

Comment:
Would you mind replacing `See commit message for further details.` with a short version of the actual description? The current approach is rather unusual and some Git users may not even be expert enough about Git to quickly get to that text, when needed. Thank you!

Reply:
I appreciate your thought, but after looking at it again I decided that it’s clear enough without the full explanation. A couple of lines above, `status = getattr(parsed, 'status', None)` obviously might produce a `None`. If anyone cares enough to notice _that_ in their combination of feedparser and Python versions `None` is not possible, then the test matrix will fail them if they try to remove this branch of the `if` statement. They will also see the commit message, if they know where to look, so I removed that sentence eventually.